tests/features/author-delegated-grant.spec.ts

```diff
@@ -62,7 +63,82 @@ export function testDelegatedGrantScenarios(): void {
       await dwn.close();
     });
 
-    it('should only allow correct entity invoking a delegated grant to write', async () => {
+    describe('RecordsWrite.parse()', async () => {
+      it('should throw if a message invokes a author-delegated grant (ID) but the author-delegated grant is not given', async () => {
+        const alice = await TestDataGenerator.generatePersona();
+        const bob = await TestDataGenerator.generatePersona();
+
+        // Alice grants Bob to write as her for the chat protocol
+        const scope: PermissionScope = {
+          interface : DwnInterfaceName.Records,
+          method    : DwnMethodName.Write,
+          protocol  : 'chat'
+        };
+        const grantToBob = await PermissionsGrant.create({
+          delegated   : true, // this is a delegated grant
+          dateExpires : Time.createOffsetTimestamp({ seconds: 100 }),
+          description : 'Allow Bob to write as me in chat protocol',
+          grantedBy   : alice.did,
+          grantedTo   : bob.did,
+          grantedFor  : alice.did,
+          scope,
+          signer      : Jws.createSigner(alice)
+        });
+
+        // Bob creates a chat message invoking the delegated grant (ID) but does not include the author-delegated grant (we remove it below)
+        const recordsWrite = await RecordsWrite.create({
+          signer         : Jws.createSigner(alice),
```

Comment:
Sanity: should the signer be bob?

Reply:
You are right, should be fixed like the other test!